client/src/Votes.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+
+function Votes() {
+  const [Like, setLike] = useState(0);
+
+  const handleLike = () => {
+    setLike(Votes + 1);
```

Comment:
I think this is a typo; this is meant to be `setLike(like+1)`? Another way to do this is `setLike((currentValue) => currentValue+1)`